src/editor-plugin/plugin.js

```diff
@@ -65,6 +65,17 @@ const EditorPlugin = () => {
 				help={ __( 'Content warnings do not change the content on your site, only in the fediverse.', 'activitypub' ) }
 			/>
 
+			<RangeControl
+				label={ __( 'Maximum Image Attachments', 'activitypub' ) }
+				value={ meta?.activitypub_max_image_attachments || window._activityPubOptions?.maxImageAttachments || 4 }
+				onChange={ ( value ) => {
+					setMeta( { ...meta, activitypub_max_image_attachments: value } );
+				} }
+				min={ 0 }
+				max={ 10 }
```

Comment:
Not sure if we ever thought about a max number of images/attachments that should be allowed? Should this be even limited?

Reply:
...if a user wants to mess up their posts...